src/filter/validation.schema.ts

```diff
@@ -78,7 +78,7 @@ export const schema = {
           additionalProperties: false,
           properties: {
             $regex: {
-              type: 'string',
+              type: ['string', 'boolean'],
```

Comment:
@maximegagnonvooban I don't think `$regex` can contain `boolean`.

Reply:
Yes, you are right. I will delete it